randovania/games/super_metroid/pickup_database/pickup-database.json

```diff
@@ -341,7 +428,12 @@
             "pickup_category": "energy_tank",
             "broad_category": "life_support",
             "model_name": "Reserve Tank",
-            "offworld_models": {},
+            "offworld_models": {
+                "am2r": "sItemEnergyTank",
+                "prime1": "Energy Tank",
+                "prime2": "EnergyTank",
+                "dread": "item_energytank"
```

Comment:
if it's desirable to distinguish between etanks and reserve tanks, echoes can use EnergyTankSmall and dread can use energy parts

Reply:
Was not aware those existed. I assume prime1 doesn't have an equivalent there?

Reply:
not yet, no